Client/Application/AppDelegate.swift

```diff
@@ -30,71 +28,30 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var orientationLock = UIInterfaceOrientationMask.all
     lazy var profile: Profile = BrowserProfile(localName: "profile",
                                                syncDelegate: UIApplication.shared.syncDelegate)
+    private let log = Logger.browserLogger
     private var shutdownWebServer: DispatchSourceTimer?
-    private var telemetry: TelemetryWrapper?
-    private var adjustHelper: AdjustHelper?
     private var webServerUtil: WebServerUtil?
     private var appLaunchUtil: AppLaunchUtil?
-
     func application(_ application: UIApplication,
                      willFinishLaunchingWithOptions
                      launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-
         log.info("startApplication begin")
 
-        appLaunchUtil = AppLaunchUtil()
-        appLaunchUtil?.setUpAppLaunchDependencies()
-
         self.window = UIWindow(frame: UIScreen.main.bounds)
 
-        telemetry = TelemetryWrapper(profile: profile)
-
-        // Initialize the feature flag subsytem.
-        // Among other things, it toggles on and off Nimbus, Contile, Adjust.
-        // i.e. this must be run before initializing those systems.
-        FeatureFlagsManager.shared.initializeDeveloperFeatures(with: profile)
-        FeatureFlagUserPrefsMigrationUtility(with: profile).attemptMigration()
-
-        // Migrate wallpaper folder
-        WallpaperMigrationUtility(with: profile).attemptMigration()
-
-        // Start intialzing the Nimbus SDK. This should be done after Glean
-        // has been started.
-        initializeExperiments()
-
-        ThemeManager.shared.updateProfile(with: profile)
+        appLaunchUtil = AppLaunchUtil(profile: profile)
+        appLaunchUtil?.setUpPreLaunchDependencies()
 
         // Set up a web server that serves us static content. Do this early so that it is ready when the UI is presented.
         webServerUtil = WebServerUtil(profile: profile)
         webServerUtil?.setUpWebServer()
 
         let imageStore = DiskImageStore(files: profile.files, namespace: "TabManagerScreenshots", quality: UIConstants.ScreenshotQuality)
-
-        // Temporary fix for Bug 1390871 - NSInvalidArgumentException: -[WKContentView menuHelperFindInPage]: unrecognized selector
-        if let clazz = NSClassFromString("WKCont" + "ent" + "View"), let swizzledMethod = class_getInstanceMethod(TabWebViewMenuHelper.self, #selector(TabWebViewMenuHelper.swizzledMenuHelperFindInPage)) {
-            class_addMethod(clazz, MenuHelper.SelectorFindInPage, method_getImplementation(swizzledMethod), method_getTypeEncoding(swizzledMethod))
-        }
```

Comment:
The underlying issue this hack was added to solve has been addressed.
https://bugs.webkit.org/show_bug.cgi?id=176077